python-avd/pyavd/_eos_designs/schema/schema_fragments/defs_node_type.schema.yml

```diff
@@ -169,7 +169,11 @@ $defs:
             documentation_options:
               table: node-type-uplink-configuration
             type: str
-            $ref: "eos_designs#/keys/node_type_keys/items/keys/uplink_type"
+            valid_values:
+              - p2p
+              - port-channel
+              - p2p-vrfs
+              - lan
```

Comment:
so we end up maintaining this in two places - would it not be better to have the other one refer to here and augment it with default?

Reply:
It would be counterintuitive to ref towards the child that inherits from the parent. I think we can live with maintaining this in two places.

Reply:
or a defs without default somewhere and both reference it and add default where needed?

Reply:
A bit much to not maintain 4 strings.

Reply:
I agree with Claus; perhaps just put a comment so we don't accidentally forget in the future!

Reply:
normally we search for these things and update where needed.